build/pf9-release-image.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# This file will build k8s image. retag it and push image to dockerhub
+make quick-release-images KUBE_EXTRA_WHAT="cmd/kube-proxy" KUBE_BUILD_CONFORMANCE=n KUBE_BUILD_HYPERKUBE=n KUBE_VERBOSE=0 KUBE_BUILD_PLATFORMS=linux/amd64 KUBE_DOCKER_REGISTRY=platform9
```

Comment:
Nice to have these flags and build only what's necessary.

Reply:
However, we should add a script / make target later than helps test things locally - including run the conformance tests etc. before releasing an image.

Reply:
yeah this script is only for publishing image with teamcity. not intended to use for local development. you can just use `make quick-release-images` for local dev.